packages/terra-date-picker/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "devDependencies": {
+    "react-intl": "^2.3.0",
```

Comment:
react-intl is added as a devDependency because it is used only for jest testing just like [Overlay](https://github.com/cerner/terra-core/blob/master/packages/terra-overlay/package.json#L24) and [Demographics Banner](https://github.com/cerner/terra-core/blob/master/packages/terra-demographics-banner/package.json#L24)